Description: Skip deleted tests

Parts of the test suite are omitted from this package because they
are not DFSG-free.  Skip those parts to allow running the rest.

Author: Rebecca Palmer <rebecca_palmer@zoho.com>
Bug-Debian: https://bugs.debian.org/767387

diff --git a/utests/CMakeLists.txt b/utests/CMakeLists.txt
index 9c531de..43965ac 100644
--- a/utests/CMakeLists.txt
+++ b/utests/CMakeLists.txt
@@ -22,12 +22,8 @@ set (utests_sources
   utest_error.c
   compiler_basic_arithmetic.cpp
   compiler_displacement_map_element.cpp
-  compiler_shader_toy.cpp
   compiler_mandelbrot.cpp
   compiler_mandelbrot_alternate.cpp
-  compiler_box_blur_float.cpp
-  compiler_box_blur_image.cpp
-  compiler_box_blur.cpp
   compiler_insert_to_constant.cpp
   compiler_argument_structure.cpp
   compiler_arith_shift_right.cpp
